consensus/types/src/fork_name.rs

```diff
@@ -4,7 +4,9 @@ use ssz_derive::{Decode, Encode};
 use std::fmt::{self, Display, Formatter};
 use std::str::FromStr;
 
-#[derive(Debug, Clone, Copy, Decode, Encode, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(
+    Debug, Clone, Copy, Decode, Encode, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize,
+)]
```

Comment:
would be good to add a test that checks agreement between `ForkName::list_all` and `Ord`

Reply:
added one here https://github.com/sigp/lighthouse/pull/5531/commits/c34b26e1472a9d99e74c724529bed0468c4b2456